swagger.yaml

```diff
@@ -959,6 +992,53 @@ definitions:
             type: string
             description: 9 digits SSN
             example: '123456789'
+  PhonesResultObject:
+    properties:
+      data:
+        type: array
+        items:
+          properties:
+            id:
+              type: string
+              description: Globally unique Id for the phone numbers
+            type:
+              type: string
+              example: phone
+            attributes:
+              properties:
+                areaCode:
+                  type: string
+                  example: 541
```

Comment:
If a string can be parsed as a number or other types, it should be quoted.

Ref: https://github.com/osu-mist/style-guides/blob/master/OpenAPI.md#quoting-conventions